app/src/main/java/org/p2p/wallet/send/SendModule.kt

```diff
@@ -61,4 +64,8 @@ object SendModule : InjectionModule {
         factoryOf(::NewSendPresenter) bind NewSendContract.Presenter::class
         factoryOf(::NewSendDetailsPresenter) bind NewSendDetailsContract.Presenter::class
     }
+
+    private fun Module.initDataLayer() {
+        factoryOf(::RecipientsDatabaseRepository)
```

Comment:
interface needed

Reply:
for what?

Reply:
for Repository
all our repositories have interfactes
e.c. org.p2p.wallet.history.repository.local.TransactionDetailsLocalRepository